packages/create-webpack-app/bin/cli.js

```diff
@@ -0,0 +1,55 @@
+import { Command } from "commander";
```

Comment:
Can we use the `webpack-cli` package? It also offers utilities like logger. Checkout the [CLI tests](https://github.com/webpack/webpack-cli/blob/master/test/api/CLI.test.js) for more info.